gptel-anthropic.el

```diff
@@ -277,35 +277,30 @@ TOOL-USE is a list of plists containing tool names, arguments and call results."
                     (goto-char (previous-single-property-change
                                 (point) 'gptel nil (point-min)))
                     (not (= (point) prev-pt)))
-          ;; HACK Until we can find a more robust solution for editing
-          ;; responses, ignore prompts containing only whitespace, as the
-          ;; Anthropic API can't handle it.  See #452, #409, #406, #351 and #321
-          ;; We check for blank prompts by skipping whitespace and comparing
-          ;; point against the previous.
+          ;; XXX update for tools
```

Comment:
What does the "XXX" tag mean?

Reply:
XXX = Don't commit to master without finishing